portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,63 @@
-<!DOCTYPE html>
+ <!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+
+    <title>Minolta's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+
+   <!--<div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
-    </div>
+    </div>-->
+
+    <!--========= Header ========-->    
+    <header>
+        <div class="container">
+            <div class="row d-flex justify-content-center">
+                <div class="col-md-12">
+                <nav class="nav-menu">
+                    <ul>
+                    <li><a href="#!">About</a></li> 
+                    <li><a href="#!">Projects</a></li> 
+                    <li><a href="#!">Resume</a></li> 
+                    <li><a href="#!">Blog</a></li> 
+                    <li><a href="#!">Contact</a></li> 
+                    </ul>
+                </nav>
+                </div>
+            </div>
+        </div>
+    </header> <!--End Header-->
+
+    <!--========= Hero Section ========-->
+    <section id="hero" class="d-flex justify-content-center align-items-center">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+                    <div><h1>Minolta Ndlovu</h1></div>
+                    <di><p>A front-end developer from Bulawayo, Zimbabwe</p></div>
+                    <div><a href="#!">About me</a></div>
+                </div>
+            </div>
+        </div>
+    </section>
+   
+    <!--========= About Me Section ========-->
+    <section></section>
+
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
```

Comment:
Split this line too

https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping